lib/httparty.rb

```diff
@@ -200,6 +200,19 @@ def read_timeout(t)
       default_options[:read_timeout] = t
     end
 
+    # Allows setting a default write_timeout for all HTTP calls in seconds
+    # Supported by Ruby > 2.6.0
+    #
+    #   class Foo
+    #     include HTTParty
+    #     write_timeout 10
+    #   end
+    def write_timeout(t)
+      raise ArgumentError, 'write_timeout must be an integer or float' unless t && (t.is_a?(Integer) || t.is_a?(Float))
```

Comment:
Could extract it to method and refactor other timeout methods as well, please?
Something like `validate_timeout_argument(timeout)` or `check_timeout_argument(timeout)`